pom.xml

```diff
@@ -60,8 +60,8 @@
 
     <properties>
         <!-- Hazelcast branch to download and compile with -->
-        <hazelcast.git.repo>hazelcast</hazelcast.git.repo>
-        <hazelcast.git.branch>master</hazelcast.git.branch>
+        <hazelcast.git.repo>mdogan</hazelcast.git.repo>
```

Comment:
Is this correct?

Reply:
yes, current cp needs custom branches before the merge.

Reply:
That's the only way for now, since client protocol and Hazelcast have a cyclic dependency to each other.